physical/datasource.go

```diff
@@ -109,5 +109,5 @@ func (dsb *DataSourceBuilder) Materialize(ctx context.Context, matCtx *Materiali
 }
 
 func (dsb *DataSourceBuilder) Metadata() *metadata.NodeMetadata {
-	return metadata.NewNodeMetadata(dsb.Cardinality)
+	return metadata.NewNodeMetadata(dsb.Cardinality, "")
```

Comment:
different approach here than everywhere else (why is that so?) :
`return metadata.NewNodeMetadata(cardinality,  octosql.NewVariableName(""))`

Reply:
Fixed.